frontends/web/src/routes/account/send/send.tsx

```diff
@@ -637,11 +638,9 @@ class Send extends Component<Props, State> {
                         disabled={!this.getValidTxInputData() || !valid || isUpdatingProposal}>
                         {t('send.button')}
                       </Button>
-                      <ButtonLink
-                        secondary
-                        to={`/account/${code}`}>
+                      <BackButton secondary>
```

Comment:
I am confused, `secondary` is hardcoded in BackButton and this `secondary` here is unused I think. Why is nothing complaining about unused props?

Reply:
TButton accepts this prop. This is a mistake, I'll update once we decide if BackButton has a future or is just temp.

Reply:
Imo it has a future just so the `navigate(-1)` onclick thing does not have to be repeated everywhere. then we could also easily hide it completely on Android if we wanted to.

Reply:
ack 👍